processes/org-monthly-meetings/minutes/2019-07-Minutes.md

```diff
@@ -0,0 +1,254 @@
+
+Event went really well, and had a good traction.
+
+SADLY this event can be **closed** until next year
+
+
+## Handbook for organisers
+
+**Owner:** @amaliacardenas
+
+**Status:**
+No updates.
+Keep into account the organizer expectations.
+
+This is topic is **ongoing** and to be included in the next meeting.
+
+
+## Instagram account
+
+**Owner:** @raquelalvear
+
+**Status:**
+Account is up and running!
+Documentation on how to use it is missing.
+Can get help and inspiration from ThoughtWorks office manager.
+
+This topic is **ongoing** and to be included in the next meeting
+
+
+## Adding a community manager
+
+**Owner:** @loriking
+
+**Status:**
+To be merged into the instagram account
+
+
+
+# New initiatives
+
+## communicate the minutes of our org meetings in the Bcn channel to promote interest in the org and have some initiative from our members
+
+**Owner:**
+
+**Status:**
+This item was not discussed in the meeting.
+
+
+## Additional workshops?
+
+**Owner:** @raquelalvear
+
+**Status:**
+The scope of this initative is for regular workshops.
+Combining specific workshops and extra regular workshops might be to much.
+Could we incour in couch burnout?
+It is an opportunity for doing it in different languages (spanish / catalan), maybe with this we can rely on different coaches.
+We can invest in promoting what a coach is and how it can be useful for organizations to promote.
+
+We need to work further on define what is what we want. We need to gather intel from coaches, students, hosts.
+Survey time for mid september?. Is it ok to use the codebar mailing list for the surveys?
+
+This topic is **ongoing** and should be included in the next meeting.
+
+
+## a kit for coach so that they can explain to their coworker why coaching at codebar is super cool ( came from a chat with Alyssa from typeform to how to foment their employee to coach at codebar when it will be at typeform)
+
+**Owner:**
+
+**Status:**
+This item was not discussed in the meeting.
+
+
+## BCN codebar role in building LGBTQIA + tech community
+
+**Owner:** @daverick
+
+**Status:**
+
+We should push this forward and promote diversity in different companies and other spaces, in a collaborative manner with other organizations already doing this.
+There is a Donate button in the codebar page, it is possible to communicate with Codebar the donations that are donated for the barcelona chapter for this purpose.
+
+This topic is **ongoing** and should be included in the next meeting.
+
+
+## Working with some student on making better the feedback from student to coach
+
+**Owner:**
+
+**Status:**
+
+Possibility to do a PR on improving Codebar page in regards of feedback?
+
+
+## reuse TW computing fundamentals
+
+**Owner:**
+
+**Status:**
+
+TW computer fundamentals is for advanced topics. We do not know if the students would be interested in it.
+
+Facilitator sends this topic to the **parking lot**
+
+
+## better pairing support
+
+**Owner:** @jmbriano & @amaliacardenas
```

Comment:
I'm not 100% sure but I think @daverick was co-owning this one too

Reply:
yes
```suggestion
**Owner:** @jmbriano & @amaliacardenas & @daverick
```